subhub/tests/unit/test_payment_calls.py

```diff
@@ -136,6 +136,38 @@ def test_create_subscription_with_valid_data():
     g.subhub_account.remove_from_db("validcustomer")
 
 
+def test_subscribe_customer_existing(create_customer_for_processing):
```

Comment:
Probably want to add a test to show what happens when you cancel a subscription that was already cancelled. I would expect that in that case, the command would still go through ok, even though nothing needs to be done. If an error is thrown, then it should indicate that it was already cancelled.

Reply:
Done.